src/main/java/spoon/reflect/visitor/filter/OverriddenMethodFilter.java

```diff
@@ -20,25 +20,16 @@
 import spoon.reflect.declaration.CtMethod;
 import spoon.reflect.declaration.CtType;
 import spoon.reflect.visitor.Filter;
-import spoon.reflect.visitor.chain.CtQueryStep;
-import spoon.reflect.visitor.chain.CtConsumer;
+import spoon.reflect.visitor.chain.CtQueryImpl;
 import spoon.reflect.visitor.chain.CtQuery;
 
 /**
  * Gets all overridden method from the method given.
  */
```

Comment:
Interesting idea to extend OverridenMethodFilter from the CtQueryImpl. But rollback all the changes (keep trunk version), becous your implementation does nothing and my will not work with your changes.